docs/reference/azpol.md

```diff
@@ -0,0 +1,295 @@
+
+## Prevent Public IP based services
+
+Most of the Azure Platform-as-a-service (PaaS) services are created with a public IP address assigned to them. This is a good option for developers who want to quickly get started with these services. Public endpoint accelerates learning curve and is ideal when developing pilots and small-scales Proof Of Concept (PoC) implementations.
+
+However, when these pilots/PoCs make transition to production-ready enterprise applications, their use of public IP addresses is sometimes overlooked.
+
+Production workloads using public IPs without proper security measures in place can increase security risks. Malicious actors can potentially use public IP as a gateway to launch an attack. Many enterprise compliance policies do not allow use of public IP just to avoid exposure to such security risks.
+
+There is a custom ESLZ policy on denying creation of public IP address which prevents pubic IP getting created in a scope targeted by policy. Enterprises can easily prevent Virtual Machines (VMs) getting created with public IP using this policy.
+
+Similarly there is a custom ESLZ Policy Initiative (also referred as PolicySet) which  helps enterprises prevent Azure services getting created with a public IP address in the first place. Following key Azure services are prevented ever getting created with a Public IP address.
+
+| Azure Service|
+|:-------------|
+Azure CosmosDB|
+Azure MariaDB|
+Azure MySQL|
+Azure PostgreSql|
+Azure KeyVault|
+Azure SQL Database|
+Azure Storage|
+Azure Kubernetes Service|
+
+## Enforce audit and log information collection
+
+Lack of auditing and diagnostics information at granular level can impact operational practices. Incomplete audit information makes it difficult to correlate logs from multiple Azure services and form a coherent debugging experience.
+
+It is desirable that once Azure services are provisioned, they provide detailed information about Azure platform they interact with.
+Such information can be broadly divided into logs and metrics. Each Azure service can be further categorized into its sub-components (e.g. An Azure Public IP resource has `DDoSProtectionNotifications`, `DDoSMitigationReports` and `DDoSMitigationFlowLogs` as its sub-components. Collecting diagnostic information at these sub-categories can greatly enhance auditing and debugging experience.
+
+ ESLZ deploys a custom Policy Initiative to enforce logs and metrics collection at a deeper level which helps enterprises gather logs and metrics per Azure Service as shown below.
+
+| Azure Service  | Log Categories           |Metrics        |
```

Comment:
Would be great if we could replace this table with a general statement. That metrics are enabled accross all services